src/styles/components/_hamburger.scss

```
@@ -0,0 +1,113 @@
+.hamburger {
+  position: absolute;
+  left: rem(16px);
+  width: 100%;
+
+  @include respond-above('md') {
+    display: none;
+  }
+
+  &__container {
+    user-select: none;
+
+    > span {
+      position: relative;
+      display: block;
+      width: rem(24px);
+      height: rem(3px);
+      border-radius: rem(3px);
+      margin-bottom: rem(5px);
+
+      background: $dark;
+
+      transform-origin: rem(4px 0px);
+
+      @include transition(transform, 0.55s, ease);
+      @include transition(cubic-bezier(0.77, 0.2, 0.05, 1) opacity, 0.5s, ease);
```

Do you think we should group them into one line? Any better ideas? @LilBank @Raksani 

```suggestion
      @include transition(transform 0.55s ease, cubic-bezier(0.77, 0.2, 0.05, 1) opacity 0.5s ease);
```

---

One line would be better for me.